TLM/TLM/UI/SubTools/SpeedLimits/Overlay/SpeedLimitsOverlay.cs

```diff
@@ -452,211 +458,207 @@ private struct CachedSegment {
 
                 this.cachedVisibleSegmentIds_.Add(
                     new CachedSegment {
+                        isDirty = true,
                         id_ = (ushort)segmentId,
                         center_ = segment.GetCenter(),
                     });
             } // end for all segments
         }
 
+        private DrawEnv env_;
+
         /// <summary>
-        /// Render speed limit handles one per segment, both directions averaged, and if the speed
-        /// limits on the directions don't match, extra small speed limit icons are added.
+        /// Draw speed limit signs (only in GUI mode).
+        /// NOTE: This must be called from GUI mode, because of GUI.DrawTexture use.
+        /// Render the speed limit signs based on the current settings.
         /// </summary>
-        /// <param name="segmentId">Seg id.</param>
-        /// <param name="segCenter">Bezier center for the segment to draw at.</param>
-        /// <param name="camPos">Camera.</param>
-        /// <param name="args">Render args.</param>
-        private bool DrawSpeedLimitHandles_PerSegment(ushort segmentId,
```

Comment:
This wrapper method became superfluous after refactoring so removed it.

Reply:
I couldn't find any references to it so hopefully it doesn't break anything heh.